introduction_to_amazon_algorithms/jumpstart-foundation-models/text-generation-benchmarking/benchmarking/payload.py

```diff
@@ -0,0 +1,5 @@
+def create_test_payload(input_words: int, output_tokens: int):
+    return {
+        "inputs": "I believe the meaning of life is to " * (input_words // 8),
```

Comment:
such a profound question. 

are there other parameters we'd want to toggle? is there any need to generate the text dynamically?

Reply:
This benchmarking is performed on a fixed payload. By controlling the exact payload and concurrency load, we can normalize on good environment variable configurations to use when deploying an instance. That's not to say that dynamic load testing is not helpful, just not the desired output of this script.